chart/deploy-cert-manager/templates/01-deploy-cert-manager.yaml

```diff
@@ -0,0 +1,42 @@
+apiVersion: v1
```

Comment:
There is alot of duplicate values used here (such as `ibm-cert-manager-operator` `ibm-cert-manager-catalog` and `openshift-marketplace`). Would it not be easier to store them as helm values as use helm templating to substitute the values instead of duplicating them?